### PR TITLE
Add assertions.xml for fscanf() and one test

### PR DESCRIPTION
**Pull request by padmashree9110**
Adds
1>1-1.c
2>assertions.xml

Signed-off-by: Mandri, Padmashree <padmashree.mandri@intel.com>

### CODE REVIEW

File: testcases/open_posix_testsuite/conformance/interfaces/fscanf/1-1.c [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+ *  -write 3 integers into the file
+ *  -close the file
+ *  -open the same file in read mode
+ *  -read only 2 integers from the file using fscanf() while giving 3 pointer
+ *   arguments,
+ *  -third integer value will be ignored
+ *  -check the 2 read integer values
+ */
+
+#define _XOPEN_SOURCE 700
```
Comment (pevik) [inline testcases/open_posix_testsuite/conformance/interfaces/fscanf/1-1.c]: FYI:
https://patchwork.ozlabs.org/patch/1008664/#2047324
http://lists.linux.it/pipermail/ltp/2018-December/010211.html
http://lists.linux.it/pipermail/ltp/2018-December/010231.html

**Comment by metan-ucw** [comment]
@padmashree9110 if you are going to work on LTP it's probably a good idea to subscribe to the mailing list and read messages related to your tests there.

File: testcases/open_posix_testsuite/conformance/interfaces/fscanf/1-1.c [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+				errno);
+		remove_file(file_str);
+		exit(PTS_UNRESOLVED);
+	}
+
+	errno = 0;
+	#pragma GCC diagnostic push
+	#pragma GCC diagnostic ignored "-Wformat-extra-args"
+	read_values = fscanf(in_fp, "%d %d", &var1, &var2, &var3);
+	#pragma GCC diagnostic pop
```
Comment (metan-ucw) [inline testcases/open_posix_testsuite/conformance/interfaces/fscanf/1-1.c]: Please never ever add pragmas like that into the tests, these break compilation on many systems.

File: testcases/open_posix_testsuite/conformance/interfaces/fscanf/1-1.c [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+#include <errno.h>
+#include <string.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <limits.h>
+#include "posixtest.h"
+
+#define BUF_SIZE 1024
+#define TNAME "fscanf/1-1.c"
+#define FNAME "/tmp/in_file"
```
Comment (metan-ucw) [inline testcases/open_posix_testsuite/conformance/interfaces/fscanf/1-1.c]: The files in tmp should have reasonably unique name, most tests use pid to make names unique, it's also good idea to include name of the test in the temporary file name such as "fscanf_1-1_$pid".

File: testcases/open_posix_testsuite/conformance/interfaces/fscanf/1-1.c [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+#define VAR1 123
+#define VAR2 456
+#define VAR3 789
+
+static void remove_file(char *file_name)
+{
+	errno = 0;
+	(void)unlink(file_name);
+}
+
```
Comment (metan-ucw) [inline testcases/open_posix_testsuite/conformance/interfaces/fscanf/1-1.c]: Why do we need this ugly wrapper around unlink?

File: testcases/open_posix_testsuite/conformance/interfaces/fscanf/1-1.c [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+
+int main(void)
+{
+	int var1, var2, var3;
+	int ret, read_values;
+	FILE *in_fp;
+	char file_str[BUF_SIZE];
+
+	snprintf(file_str, BUF_SIZE, "%s-%d", FNAME, getpid());
```
Comment (metan-ucw) [inline testcases/open_posix_testsuite/conformance/interfaces/fscanf/1-1.c]: Ah, it's made unique here, then there is no point in keeping the static part of the string in a macro, just move it to the formatting string instead.

File: testcases/open_posix_testsuite/conformance/interfaces/fscanf/1-1.c [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+	if (ret < 0) {
+		printf(TNAME " Error in writing to the file\n");
+		remove_file(file_str);
+		exit(PTS_UNRESOLVED);
+	}
+
+	errno = 0;
+	ret = fclose(in_fp);
+	if (ret != 0) {
+		printf(TNAME " Error in closing the file descriptor, errno = %d\n",
```
Comment (metan-ucw) [inline testcases/open_posix_testsuite/conformance/interfaces/fscanf/1-1.c]: This is not file descriptor at all, it's a FILE abstraction on the top of it and many things can go wrong there, so this should read just "error in closing the file"

**Pull request closed without merging**